web/src/drafts.ts

```diff
@@ -237,6 +237,22 @@ export function update_compose_draft_count(): void {
     }
 }
 
+export function update_compose_draft_count_in_popover(): void {
+    const $drafts_option = $(".view_contextual_drafts");
+    const $count_ele = $drafts_option.find(".unread_count");
+    if (!compose_state.has_full_recipient()) {
+        $count_ele.hide();
+        return;
+    }
+    const compose_draft_count = Object.keys(filter_drafts_by_compose_box_and_recipient()).length;
+    if (compose_draft_count > 0) {
+        $count_ele.text(compose_draft_count > 99 ? "99+" : compose_draft_count);
+        $count_ele.show();
+    } else {
+        $count_ele.hide();
+    }
+}
```

Comment:
How does this differ from the previous function? It seems kinda a duplicate... is the issue just that different element names? If so, we should just make `$count_ele` a parameter, maybe calling it `$count_element`.

Reply:
The issue was that one unread counter (the one one above the compose box send button), displays the number between two brackets `()` while the other unread counter (the one in the popover menu) is similar to the left sidebar one.

Reply:
I matched the structure of the both the count elements, and just removed the brackets `()` from the count element in the popover drafts count.

```
<span class="compose-drafts-count-container"><span class="unread_count compose-drafts-count"></span></span>
```

Reply:
Here, the only purpose of `.compose-drafts-count-container` container is to match the structure expected by the `drafts.update_compose_draft_count()` method.

Reply:
Sounds good!